### PR TITLE
Danse's Third Party Ships: Part 1, the Southern Fleet Administration, TCFL, and Einstein Engines

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Cleared with the lore team. The idea is that this is a neutral/wildcard SFA ship that escaped destruction by fleeing to Biesel, but now doesn't know what to do. There's also a friendly TCFL peacekeeper ship, carrying two detainees, and an unfriendly (but explicitly non-hostile!) Einstein Engines ship that's snooping around.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

File: maps/away/ships/sol.dm [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	dock_target = "sfa_shuttle"
+	current_location = "nav_hangar_sfa"
+	landmark_transition = "nav_transit_sfa"
+	range = 1
+	fuel_consumption = 2
+	logging_home_tag = "nav_hangar_sfa"
+	defer_initialisation = TRUE
+	mothershuttle = "SFA Patrol Ship"
+
+/obj/effect/shuttle_landmark/sfa_shuttle/hangar
```
Comment (NonQueueingMatt) [inline maps/away/ships/sol.dm:114]: Add `movable_flags = MOVABLE_FLAG_EFFECTMOVE` to this landmark.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by TheDanseMacabre** [comment]
100% complete and ready for immediate speedmerge.

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]
The EE ship MAY need an antag type assigned to the spawners. We'll see how it goes.

**Pull request merged**